build/Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM docker.io/golang:latest AS builder
+
 ENV OPERATOR_PATH=/gcp-project-operator
 COPY . ${OPERATOR_PATH}
 WORKDIR ${OPERATOR_PATH}
 RUN make gobuild
 
 FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
-ENV OPERATOR_PATH=/gcp-project-operator \
-    OPERATOR_BIN=gcp-project-operator
+ENV OPERATOR_BIN=gcp-project-operator \
+    OPERATOR_PATH=/${OPERATOR_BIN}
+ARG  OPERATOR_BASE_PATH=/gcp-project-operator/build/_output/bin
 
 WORKDIR /root/
-COPY --from=builder /gcp-project-operator/build/_output/bin/${OPERATOR_BIN} /usr/local/bin/${OPERATOR_BIN}
+COPY --from=builder ${OPERATOR_BASE_PATH}/gcp-project-operator /usr/local/bin/gcp-project-operator
+COPY --from=builder ${OPERATOR_BASE_PATH}/monitoring /usr/local/bin/monitoring
```

Comment:
I'd personally prefer the "one process per container" approach over putting the operator and a separate monitoring binary to run in the same container.

Reply:
This was an option but this is the best way to tie the version of the operator to the monitoring binary.
I can also do the separation  but that involves creating another quay repo.

Reply:
Having it in the same container image doesn't mean the container necessarily runs both. If it is a significant overhead to have a separate image, I'd say we can do it like this. However, I think we need to add another `container` to the operator [deployment](https://github.com/openshift/gcp-project-operator/blob/master/deploy/operator.yaml) for that.